src/mono/wasm/runtime-test.js

```diff
@@ -166,7 +167,7 @@ while (args !== undefined && args.length > 0) {
 	} else if (args [0].startsWith ("--working-dir=")) {
 		var arg = args [0].substring ("--working-dir=".length);
 		working_dir = arg;
-		args = args.slice (1);
+        args = args.slice (1);
```

Comment:
I believe this was correct to use tabs before.

Reply:
addressed